v4/export/sql_type.go

```diff
@@ -121,17 +223,12 @@ func (s *SQLTypeString) ReportSize() uint64 {
 }
 func (s *SQLTypeString) ToString() string {
 	if s.Valid {
-		return fmt.Sprintf(`'%s'`, escape(s.String))
+		return fmt.Sprintf(`'%s'`, globalEscape.Escape(s.String))
```

Comment:
why not return `globalEscape.Escape(s.String)` directly?

Reply:
because we need to use quotation marks ('') to annotate string type sql variable.